content/contributing/documentation/rst_guidelines.rst

```diff
@@ -1,162 +1,1365 @@
+                       public static void main(String[] args) {}
+                   }
+
+.. _contributing-rst-code:
+
+Code tabs
+---------
+
+Code tabs are essentially :ref:`group tabs <contributing-rst-group-tabs>` that treat
+the content as a :ref:`code block <contributing-rst-code-blocks>`. The `code-tab`
+markup is used to define a code tab. Just as for the `code-block` markup, the language defines the
+syntax highlighting of the tab. If set, the label is used instead of the language for grouping tabs.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - .. tabs::
+
+          .. code-tab:: c++ Hello C++
+
+             #include <iostream>
+
+             int main() {
+                 std::cout << "Hello World";
+                 return 0;
+             }
+
+          .. code-tab:: python Hello Python
+
+             print("Hello World")
+
+          .. code-tab:: javascript Hello JavaScript
+
+             console.log("Hello World");
+
+   * - .. code-block:: text
+
+          .. tabs::
+
+             .. code-tab:: c++ Hello C++
+
+                #include <iostream>
+
+                int main() {
+                    std::cout << "Hello World";
+                    return 0;
+                }
+
+             .. code-tab:: python Hello Python
+
+                print("Hello World")
+
+             .. code-tab:: javascript Hello JavaScript
+
+                console.log("Hello World");
+
+.. _contributing-rst-cards:
+
+Cards
+=====
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - .. cards::
+
+          .. card:: Documentation
+             :target: ../documentation
+             :tag: Step-by-step guide
+             :large:
+
+             Use this guide to acquire the tools and knowledge you need to write documentation.
+
+          .. card:: Content guidelines
+             :target: content_guidelines
+
+             List of guidelines and trips and tricks to make your content shine at its brightest!
+
+          .. card:: RST guidelines
+             :target: rst_guidelines
+
+             List of technical guidelines to observe when writing with reStructuredText.
+
+   * - .. code-block:: text
+
+          .. cards::
+
+             .. card:: Documentation
+                :target: ../documentation
+                :tag: Step-by-step guide
+                :large:
+
+                Use this guide to acquire the tools and knowledge you need to write documentation.
+
+             .. card:: Content guidelines
+                :target: content_guidelines
+
+                List of guidelines and trips and tricks to make your content shine at its brightest!
+
+             .. card:: RST guidelines
+                :target: rst_guidelines
+
+                List of technical guidelines to observe when writing with reStructuredText.
+
+.. _contributing-rst-document-metadata:
+
+Document metadata
+=================
+
+| Sphinx supports document-wide metadata markups that specify a behavior for the entire page.
```

Comment:
What is Sphinx? This is the first mention of it and has no explanation of what it is.

Reply:
It's the documentation generator that we use. It converts RST into other formats such as HTML (amongst other things). You can check the Wikipedia article https://en.wikipedia.org/wiki/Sphinx_(documentation_generator) for a more detailed explanation. I added the link in the doc as well :slightly_smiling_face: